From 6e4d018a09f0c0c0571e4728483be4bb9cf2ea52 Mon Sep 17 00:00:00 2001
From: Ryan Brown <rybrown@redhat.com>
Date: Wed, 10 Sep 2014 09:46:50 -0400
Subject: [PATCH] Add heat-keystone-setup-domain to sphinx conf.py

The man page for heat-keystone-setup-domain was not being generated
because it was not included in the sphinx conf.py.

Change-Id: Ic1c2025fc7f61e3c37906b42bd84e1df893920fc
---
 doc/source/conf.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/doc/source/conf.py b/doc/source/conf.py
index f2e30c6..ce7b52b 100644
--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -417,6 +417,9 @@ man_pages = [
     ('man/heat-keystone-setup', 'heat-keystone-setup',
      u'Script which sets up keystone for usage by Heat',
      [u'Heat Developers'], 1),
+    ('man/heat-keystone-setup-domain', 'heat-keystone-setup-domain',
+     u'Script which sets up a keystone domain for heat users and projects',
+     [u'Heat Developers'], 1),
     ('man/heat-manage', 'heat-manage',
      u'Script which helps manage specific database operations',
      [u'Heat Developers'], 1),
